docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
+
+```
+$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
+[... fetching...]
+Plugin flocker:v0.1 loaded and registered with extension points:
+* volumes
+Plugin flocker activated.
+```
+
+This is really just syntactic sugar for the following:
+
+```
+$ docker run -d -e $ARGS \
+	-v /var/run/docker-plugins/flocker.sock:/var/run/plugin.sock \
```

Comment:
On the host side, I think inside `/var/run/docker/plugins/<id>` would be the place to put that.
In the container, /run or /var/run seems like the right thing.